examples/pip_parse/BUILD.bazel

```diff
@@ -55,6 +56,11 @@ alias(
     actual = entry_point("yamllint"),
 )
 
+entry_point_binary(
+    name = "yamllint_binary",
+    pkg = "yamllint",
+)
```

Comment:
Since the basic feature is to allow additional dependencies, please provide an example where it does so. Otherwise it isn't clear what the difference is between these two, and its harder for readers to see code reflective of their case.